samples/bluetooth/peripheral_hids_keyboard/src/main.c

```diff
@@ -140,17 +156,38 @@ static struct keyboard_state {
 	u8_t keys_state[KEY_PRESS_MAX];
 } hid_keyboard_state;
 
+#if CONFIG_NFC_OOB_PAIRING
+static struct k_work adv_work;
+static struct bt_le_oob oob_local;
+
+static u8_t ndef_msg_buf[256]; /* Buffer used to hold an NFC NDEF message. */
+
 
+static void advertising_start(struct k_work *work)
+#else
 static void advertising_start(void)
+#endif
 {
-	int err = bt_le_adv_start(BT_LE_ADV_CONN, ad, ARRAY_SIZE(ad),
-				  sd, ARRAY_SIZE(sd));
-
+	int err;
+	struct bt_le_adv_param *adv_param = BT_LE_ADV_PARAM(
+						BT_LE_ADV_OPT_CONNECTABLE |
+						BT_LE_ADV_OPT_ONE_TIME,
+						BT_GAP_ADV_FAST_INT_MIN_2,
+						BT_GAP_ADV_FAST_INT_MAX_2);
+
+	err = bt_le_adv_start(adv_param, ad, ARRAY_SIZE(ad), sd,
```

Comment:
Ideally I think this should have a timeout to stop the advertiser if no connection is made. After a while the OOB address will have become invalid (cycled by privacy address refresh timeout).

Reply:
The privacy feature is turned off for this sample, I wouldn't make this sample even more complicated. We can demonstrate how to use privacy in other samples.